config/initializers/resque_config.rb

```diff
@@ -1,4 +1,8 @@
 require 'resque'
 
-config = ENV.fetch("RAILS_RESQUE_REDIS", "127.0.0.1:6379")
-Resque.redis = config
+# Previously, this initializer always set Resque.redis, and defaulted to '127.0.0.1:6379' if
+# RAILS_RESQUE_REDIS was not set.
+
+if ENV['RAILS_RESQUE_REDIS'] && ENV['RAILS_RESQUE_REDIS'].length
```

Comment:
It's better to write `ENV['RAILS_RESQUE_REDIS'].present?` as we in rails context with activesupport.